Project.toml

```
@@ -35,10 +35,13 @@ ZipFile = "a5390f91-8eb1-5f08-bee0-b1d1ffed6cea"
 [weakdeps]
 CopernicusMarine = "cd43e856-93a3-40c8-bc9e-6146cdce14fa"
 Reactant = "3c362404-f566-11ee-1572-e11a4b42c853"
+SpeedyWeather = "9e226e20-d153-4fed-8a5b-493def4f21a9"
+XESMF = "2e0b0046-e7a1-486f-88de-807ee8ffabe5"
 
 [extensions]
 ClimaOceanCopernicusMarineExt = "CopernicusMarine"
 ClimaOceanReactantExt = "Reactant"
+ClimaOceanSpeedyWeatherExt = ["SpeedyWeather", "XESMF"]
```

Two points here:

1. I think it should be:

   ```suggestion
   ClimaOceanSpeedyWeatherExt = "SpeedyWeather"
   ```

   since we want this extension to be triggered with `using SpeedyWeather` and not with `using XESMF`.

2. We need a compat entry for SpeedyWeather.

---

Yeap... Gotcha. Just run into this problem locally! :)